src/Telemetry/Core.php

```diff
@@ -30,9 +30,10 @@ class Core {
 	 * @var string[]
 	 */
 	private array $subscribers = [
-		Cron_Subscriber::class,
-		Opt_In_Subscriber::class,
 		Exit_Interview_Subscriber::class,
+		Last_Send_Subscriber::class,
+		Opt_In_Subscriber::class,
+		Telemetry_Subscriber::class,
 		Admin_Subscriber::class,
```

Comment:
🟠 
We should keep arrays like this in alphabetical order to make the classes easier to find in the list.